packages/kit/test/apps/basics/test/test.js

```diff
@@ -264,10 +264,10 @@ test.describe('Load', () => {
 			const payload_b = '{"status":200,"statusText":"","headers":{},"body":"Y"}';
 			// by the time JS has run, hydration will have nuked these scripts
 			const script_contents_a = await page.innerHTML(
-				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="3t25"]'
+				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="1vn6nlx"]'
 			);
 			const script_contents_b = await page.innerHTML(
-				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="3t24"]'
+				'script[data-sveltekit-fetched][data-url="/load/serialization-post.json"][data-hash="1vn6nlw"]'
```

Comment:
I missed this initially - is this change deliberate or accidental?

Reply:
@ghostdevv this change is deliberate, because the hashes has been changed, now the hashes are the same as in  `POST fetches are serialized` test case , https://github.com/sveltejs/kit/blob/master/packages/kit/test/apps/basics/test/test.js#L243